src/modules/list.js

```diff
@@ -0,0 +1,131 @@
+export default class List {
+  constructor() {
+    this.list = JSON.parse(localStorage.getItem('todo-list'));
+    if (!this.list) {
+      this.list = [];
+    }
+    this.display();
```

Comment:
✅ Well done on the implementation of the different functions. 
For this constuctor, maybe should you consider using ternary syntax to keep it simpler. 
this.list = JSON.parse(localStorage.getItem('todo-list')) || [];